src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import "./Introduce.scss";
+
+function Introduce() {
+  const navigate = useNavigate();
+
+  function HandleInput() {
+    console.log("input 태그 동작");
+  }
+
+  function HandleButton() {
+    navigate("/Product");
+  }
+  return (
+    <div className="introduce">
+      <img className="photo" alt="hambbang" src="images/IMG_7632.JPG" />
+      <div className="letters">
+        <p className="name">양회진</p>
+        <p className="number">47기</p>
+      </div>
+      <div className="inputButton">
+        <input id="input" onChange={HandleInput}></input>
+        <button id="button" onClick={HandleButton}>
+          클릭
+        </button>
+        <br />
+        <Link to="./Product">move to Product page</Link>
```

Comment:
```suggestion
        <Link to="/product">move to Product page</Link>
```
Link 컴포넌트의 to 속성에는 이동하고자 하는 페이지의 파일의 경로가 아닌 Router컴포넌트에 작성해주신 path 경로를 입력해주셔야 합니다.